pkg/golinters/gocritic.go

```diff
@@ -13,7 +13,7 @@ import (
 	"sync"
 
 	"github.com/go-critic/go-critic/checkers"
-	gocriticlinter "github.com/go-critic/go-critic/framework/linter"
```

Comment:
Whole `framework/linter` is deprecated and moved to `linter`. It still works just as a fallback. Will be removed in the future versions.